tests/test_assets.py

```diff
@@ -450,16 +457,41 @@ def test_lookup_symbol(self):
                 self.assertEqual(result.symbol, 'EXISTING')
                 self.assertEqual(result.sid, i)
 
-    @parameterized.expand(
-        build_lookup_generic_cases()
-    )
-    def test_lookup_generic(self, finder, symbols, reference_date, expected):
+    def test_lookup_symbol_from_multiple_valid(self):
```

Comment:
Glad you asked! We hammered out the logic we wanted back in Feb, but I don't remember all of the rationale. Here is a comment where it was outlined:

```
It seems that the best resolution is to use the following logic to map symbol S at time T:
UPDATE: I added the number of cases in each category

>1. If one SID exists for symbol S at time T, return that SID (happy path)

>2. If no SID exists for symbol S at time T, return the SID whose end_date is highest-but-not-over T, or 'none' if no such SID exists (happy path)

>3. If multiple SIDs exist for symbol S at time T, return the candidate SID whose start_date is highest. (200 cases)

>4. If mutliple SIDs exist for symbol S at time T, the best candidate SIDs share the highest start_date, return the SID with the highest end_date. (34 cases)

>5. If multiple SIDs exist for symbol S at time T, and the best candidate SIDs share the highest start_date and end_date, return None or throw an error. (no known cases)
```